src/explore-education-statistics-admin/src/pages/release/data/components/ApiDataSetAutoMappedTable.tsx

```diff
@@ -125,8 +128,8 @@ export default function ApiDataSetAutoMappedTable({
       </div>
       {filteredItemsChunks.length > 0 ? (
         <table
-          className="dfe-table--vertical-align-middl"
-          data-testid={`auto-mapped-table-${groupKey}`}
+          className="dfe-table--vertical-align-middle"
+          data-testid={tableId}
```

Comment:
Worth giving the table's `id` attribute this value as well as the `data-testid`, like the table in `ApiDataSetMappableTable.tsx`?  Or is it purely that the other table needs to have an id for anchor links to target it?